### DISCUSSION TITLE
which difference between wss://relay.walletconnect.org and wss://relay.walletconnect.com ?

### DISCUSSION THREAD

**Discussion by MarcoCiaramella** [WalletConnect/walletconnect-monorepo#795] [Developer Support]
I noticed that some dapps use the relay url wss://relay.walletconnect.org while the offical [guide](https://docs.walletconnect.com/2.0/quick-start/dapps/client) uses wss://relay.walletconnect.com. Which url is correct?

**Comment by chadyj** [comment]
relay.walletconnect.com is correct for v2
